### PR TITLE
Roll-the-dice application for PHP

### PR DESCRIPTION
**Pull request by svrnm**
One more for #2623 :-) PTAL

Preview: https://deploy-preview-2758--opentelemetry.netlify.app/docs/instrumentation/php/getting-started/

### CODE REVIEW

**Review by cartermp — APPROVED** [review]
Overall LGTM, will defer to @open-telemetry/php-approvers for code changes

File: content/en/docs/instrumentation/php/_index.md
Comment (brettmc) [inline content/en/docs/instrumentation/php/_index.md:49]:
```suggestion
environment variable to `true`. Using fibers with
```

File: content/en/docs/instrumentation/php/getting-started.md
Comment (brettmc) [inline content/en/docs/instrumentation/php/getting-started.md:12]:
```suggestion
You will learn how you can instrument a simple PHP application automatically,
```

**Review by brettmc — APPROVED** [review]

**Pull request merged**
